.github/workflows/ci.yml

```diff
@@ -0,0 +1,13 @@
+name: ci
+
+on:
```

Comment:
I think we can do the simpler
```yaml
on: [push, pull_request]
```
IIRC, we did the other one for `trino` to avoid duplicate builds when pushing to a branch in the same repository (which we do occasionally for things that need access to secrets). That's not a concern here, and it's possible that GitHub has fixed that anyway,